packages/teams-js/src/internal/communication.ts

```diff
@@ -178,7 +293,9 @@ export function sendAndHandleSdkError<T>(actionName: string, ...args: any[]): Pr
  */
 export function sendMessageToParentAsync<T>(actionName: string, args: any[] | undefined = undefined): Promise<T> {
   return new Promise((resolve) => {
-    const request = sendMessageToParentHelper(actionName, args);
+    // APIs with v0 represents beta changes haven't been implemented on them
+    // Otherwise, minimum version number will be v1
+    const request = sendMessageToParentHelper('v0', actionName, args);
```

Comment:
why this is v0? sorry I know you added comments on what v0 is, but I couldn't tell why it is v0 here.

Reply:
I think you got the answer in other places by reading my comment?